webview/src/webview_ios.mm

```diff
@@ -170,6 +174,7 @@ int Platform_Create(lua_State* L, dmWebView::WebViewInfo* _info)
     WKWebView *view = [[WKWebView alloc] initWithFrame:screen.bounds configuration:configuration];
 
     WebViewDelegate* navigationDelegate = [WebViewDelegate alloc];
+    navigationDelegate->m_AllowSiteAction = allowSiteAction; // added by dotGears/TrungB
```

Comment:
please follow the code style in the files you're editing.
As you see we use a "snake_case" pattern for local variables and parameters.

Reply:
Where is allowSiteAction defined? I don't see it anywhere.